### DISCUSSION TITLE
Does WalletConnect V2 work with WAGMI 9.x versions?

### DISCUSSION THREAD

**Discussion by kyleyakamoto** [wevm/wagmi#2929] [Q&A]
Right now I get errors importing the version of WalletConnect from WAGMI 0.9.6 which appear to be connecting to Wallet Connect V1

**Comment by tmm** [comment]
WalletConnect v2 only works with 0.12 and above.

**Reply by kyleyakamoto** [reply]
meaning it should work with 0.9.6, correct?

I get the following error with WAGMI 0.9.6 right now about every second:

`index.ts:140 WebSocket connection to 'wss://t.bridge.walletconnect.org/?env=browser&host=localhost%3A1234&protocol=wc&version=1' failed: `

**Reply by tmm** [reply]
No, 0.9.6 is before 0.12+.
